package-lock.json

```diff
@@ -2652,6 +2653,13 @@
 				"lodash": "^4.17.10"
 			}
 		},
+		"@wordpress/token-list": {
```

Comment:
I'm not quite sure why `package-json.lock` got updated here.

Reply:
I know what happened, `@wordpress/token-list` wasn't include in the `package.json` and it caused `package-lock.json` to be regenerated on 2nd `npm install` call...

/cc @aduth